index.html

```diff
@@ -226,76 +226,160 @@ <h1>The Parameter Registry</h1>
 
   <section>
     <h1>The Property Registry</h1>
-
     <p>
-This table summarizes the globally registered properties that can be provided
-as properties in a DID Document.
+      The following section defines the properties in a DID document.
     </p>
-
-    <table class="simple">
-      <thead>
-        <tr>
-          <th>Property</th>
-          <th>Description</th>
-        </tr>
-      </thead>
-
-      <tbody>
-        <tr>
-          <td>
-TBD
-          </td>
-          <td>
-            <div>DESCRIPTION_GOES_HERE</div>
-            <div><a href="">SPECIFICATION_GOES_HERE</a></div>
-            <pre>
-"@context": [
-  "https://www.w3.org/ns/did/v1",
-  "https://CONTEXT_GOES_HERE/v1"
-]
-            </pre>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </section>
-
-
     <section>
-      <h1>The Value Registry</h1>
-
-      <p>
-  This table summarizes the globally registered values that can be used as
-  property values in a DID Document.
-      </p>
+      <h2><dfn data-lt="publicKey" data-dfn-type="dfn" id="publicKey">publicKey</dfn></h2>
+      <p>An array of public keys. A public key is a verification method. Public keys are used for digital signatures, encryption and other cryptographic operations, which in turn are the basis for purposes such as authentication.</p>
```

Comment:
Hmm, pointing to a section of a spec (if one exists) is probably useful. Most other registries require that... and I'm expecting that is where this group will land.

Reply:
I think the registry is supposed to define the terms locally right? If we are creating a registry that just links to all kinds of other places where people will need to go to make updates, I don't think thats a good registry.

Reply:
> I think the registry is supposed to define the terms locally right?

No, that's not what most registries do at IETF or W3C do. I'm pretty sure that the group also didn't agree to that at the DID WG F2F. Most registries point to other places because the registry can't be normative (for a variety of good reasons that I can go into if you're interested, but will definitely eat up my time and I'm hoping to avoid that right now).

For example, where is the normative statement about "authentication" wrt. a DID Document made? The answer is that it must be in the DID Core spec, because that's the thing that states, normatively, how that property works... not the registry. The registry just documents stuff in a non-normative way and points to the thing that is the normative definition of how that property should work.